### DISCUSSION TITLE
Kumpulan Tutorial API KirimWA.id

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#77] [Pamerkan]
Thread ini berisi kumpulan tutorial untuk API KirimWA.id. Bagi kamu yang menulis atau membuat video tutorial tentang API KirimWA.id silahkan reply dibawah dan akan dikurasi di thread utama.

## TeknoCerdas.com

- [Integrasi Google Form dan WhatsApp Menggunakan API KirimWA.id](https://teknocerdas.com/programming/integrasi-google-form-dan-whatsapp-menggunakan-api-kirimwa-id/)
- [Membuat Bot WhatsApp Twitter dengan Javascript dan API KirimWA.id](https://teknocerdas.com/programming/membuat-bot-whatsapp-twitter-dengan-javascript-dan-api-kirimwa-id/)
- [Kirim WhatsApp dengan API KirimWA.id dan PHP](https://teknocerdas.com/programming/kirim-whatsapp-dengan-api-kirimwa-id-dan-php/)
- [Kirim WhatsApp dengan API KirimWA.id dan Node.js](https://teknocerdas.com/programming/kirim-whatsapp-dengan-api-kirimwa-id-dan-node-js/)
- [Kirim WhatsApp dengan API KirimWA.id dan cURL](https://teknocerdas.com/programming/kirim-whatsapp-dengan-api-kirimwa-id-dan-curl/)

**Comment by sigitmanto** [comment]
Mohon bantuannya mas @rioastamal ...., 

lagi praktekin tutorial yang [Kirim WhatsApp dengan API KirimWA.id dan PHP](https://teknocerdas.com/programming/kirim-whatsapp-dengan-api-kirimwa-id-dan-php) terhenti di langkah add device atau ga bisa pairing device. sudah keluar QR Codenya tapi gagal ketika scan dari aplikasi wa, baik yg business maupun wa biasa.

Ketika discan tidak terjadi apa2 dan kalo lanjut ke langkah berikutnya yaitu tes kirim pesan baru ada balasan kalo belum add device/pairing device.  

![gagal pairing](https://user-images.githubusercontent.com/83318237/153951466-dedc4c5b-dfad-4421-992d-dbee7fb59dd9.png)

**Reply by rioastamal** [reply]
@sigitmanto Apakah menggunakan Multi-device beta atau normal? Saat ini API KirimWA.id belum mendukung multi-device beta.

**Reply by sigitmanto** [reply]
Alhamdulillah sudah mas @rioastamal ... kemungkinan tadi seprti yg jenengan sampaikan masih multi-device atau kelamaan nge-scan nya.... sudah berhasil tes kirim pesan.
sekarang berhenti di langkah setelah membuat file index.php di folder api-kirimwa-webhook.

sebelumnya, dari awal saya sudah menjalankan tutorial ini di server tidak di localserver mas. mohon diarahkan mas. setelah membuat file index.php berikutnya bagaimana...

**Reply by rioastamal** [reply]
Silahkan ikuti saja referensi API yang ada di https://developer.kirimwa.id/api/v1/ untuk studi kasus yang akan dijalankan. Harusnya disitu sudah lengkap. Bisa juga mengikut tutorial diatas.

**Reply by sigitmanto** [reply]
Baik mas.... Trims sebelumnya....

**Comment by wrdigita** [comment]
halo kak, saya mau taya tentang instal diphpnya kak, apakah ada videonya? saya awam, soalnya

**Reply by mdestafadilah** [reply]
> halo kak, saya mau taya tentang instal diphpnya kak, apakah ada videonya? saya awam, soalnya

harus pake nodejs bu dan diserver VPS atau ngrook. PHP hanya consume webhook atau api yang disediakan.

**Comment by agusandiyono** [comment]
halo kak, untuk [Integrasi Google Form dan WhatsApp Menggunakan API KirimWA.id](https://teknocerdas.com/programming/integrasi-google-form-dan-whatsapp-menggunakan-api-kirimwa-id/) apakah memungkinkan mengirim gambar yang di upload dari google form. 

mengingat file gambar yang diupload ke google drive tidak terdapat filetypenya. terimakasih

**Comment by aanmochtar80** [comment]
halo kak untuk [Kirim WhatsApp dengan API KirimWA.id dan Node.js](https://teknocerdas.com/programming/kirim-whatsapp-dengan-api-kirimwa-id-dan-node-js/) saya terkendala untuk perintah "Untuk menjalankan suplai API Token dan device ID dari environment." apakah bisa di berikan video step by stepnya karena masih muncul terus eror  Something went wrong { body: '{"message":"Invalid API token."}', statusCode: 403 } terimakasih

**Comment by bonangtattoo-art** [comment]
kak ada script cek nama business wa kk
